### PR TITLE
【inplace api】batch add inpalce api paddle.abs_, paddle.addmm_, paddle.sin_ ...

### PR DESCRIPTION
**Pull request by GGBond8488**
### PR types
Others

### PR changes
 APIs

### Description
Pcard-72375
add inplace api as follow

> abs_
addmm_
cos_
cosh_
acos_
acosh_
sin_
sinh_
asin_
asinh_
tan_
atan_
atanh_
erf_
square_
expm1_

### CODE REVIEW

File: python/paddle/tensor/math.py [hunk 1]
```diff
@@ -2043,6 +2058,66 @@ def addmm(input, x, y, beta=1.0, alpha=1.0, name=None):
         return out
 
 
+@inplace_apis_in_dygraph_only
+def addmm_(input, x, y, beta=1.0, alpha=1.0, name=None):
+    """
+    Inplace version of ``scale`` API, the output Tensor will be inplaced with input ``x``.
```
Comment (jeff41404) [inline python/paddle/tensor/math.py]: Inplace version of ``addmm`` API?

**Review by chenwhql — APPROVED** [review]

File: python/paddle/tensor/math.py (hunk 1 above)
Comment (GGBond8488) [reply python/paddle/tensor/math.py]: changed to addmm

**Review by sunzhongkai588 — APPROVED** [review]
LGTM for docs

**Review by jeff41404 — APPROVED** [review]
LGTM

**Pull request merged**